### PR TITLE
xbps-src: add default build option description

### PR DESCRIPTION
**Pull request by Duncaen**
Most descriptions already just use this format anyways.

[ci skip]

### CODE REVIEW

File: common/xbps-src/shutils/show.sh [hunk 1]
```diff
@@ -129,15 +129,16 @@ show_pkg_makedepends() {
...
     [ -z "$PKG_BUILD_OPTIONS" ] && return 0
 
     source $XBPS_COMMONDIR/options.description
     msg_normal "$pkgver: the following build options are set:\n"
     for f in ${PKG_BUILD_OPTIONS}; do
-        opt="${f#\~}"
-        eval desc="\${desc_option_${opt}}"
+        local opt="${f#\~}"
+        local descref="desc_option_${opt}"
+        local desc="${!descref-Enable support for $opt}"
```
Comment (classabbyamp) [inline common/xbps-src/shutils/show.sh:141]: is `${foo-default}` different from `${foo:-default}`?

**Review by classabbyamp — APPROVED** [review]
+1 otherwise

File: common/xbps-src/shutils/show.sh (hunk 1 above)
Comment (Duncaen) [reply common/xbps-src/shutils/show.sh:141]: > Omitting the colon results in a test only for a parameter that is unset.

I guess I can change it to `:-` setting an empty description doesn't make much sense.

Comment (classabbyamp) [reply common/xbps-src/shutils/show.sh:141]: ok just checking, I'm more used to posix parameter expansion

**Pull request merged**